salt/crypt.py

```diff
@@ -593,7 +593,8 @@ def verify_master(self, payload):
         if os.path.isfile(m_pub_fn) and not self.opts['open_mode']:
             local_master_pub = salt.utils.fopen(m_pub_fn).read()
 
-            if payload['pub_key'] != local_master_pub:
+            if payload['pub_key'].replace('\n', '').replace('\r', '') != \
+                    local_master_pub.replace('\n', '').replace('\r', ''):
```

Comment:
Matter of style, but maybe use `translate()` to remove both characters in one go?
```python
            if payload['pub_key'].translate(None,'\r\n') != \
                    local_master_pub.translate(None, '\r\n'):
```